frontend/src/components/Page/Page.jsx

```diff
@@ -1,41 +1,56 @@
 import React, { useState, useEffect } from 'react';
 import { observer } from 'mobx-react';
-import classnames from 'classnames';
 
 import {
-  Container,
-  Row,
-  Col,
-  Text,
   Icon,
+  Breadcrumb,
+  BreadcrumbItem,
 } from '@dataesr/react-dsfr';
 
 import Tutorial from 'components/Tutorial/Tutorial';
 import Announcement from 'components/Notification/Announcement';
 import { useStore } from 'stores/';
 
+import Statistics from 'components/Landing/Statistics';
+import classNames from 'classnames';
 import FaqSection from './FaqSection';
-import UnderlinedTitle from './UnderlinedTitle';
 
 import styles from './page.cssmodule.scss';
 
+// Inspired from https://stackoverflow.com/a/60564620
+const getNodeText = node => {
```

Comment:
:orange_circle: Pourquoi avoir besoin de faire ça ?

Reply:
Because the title can be a react node (with bold part or break line)